diff --git a/node_modules/@mori2003/jsimgui/build/imgui.js b/node_modules/@mori2003/jsimgui/build/imgui.js
index 8af4c57..3a30c4d 100644
--- a/node_modules/@mori2003/jsimgui/build/imgui.js
+++ b/node_modules/@mori2003/jsimgui/build/imgui.js
@@ -9,18 +9,10 @@ export const Mod = {
             MainExport = await import(loaderPath);
         }
         else if (enableFreeType) {
-            MainExport = extensions
-                ? // @ts-expect-error
-                    await import("./loader-freetype-extensions.js")
-                : // @ts-expect-error
-                    await import("./loader-freetype.js");
+            MainExport = await import("./loader-freetype.js");
         }
         else {
-            MainExport = extensions
-                ? // @ts-expect-error
-                    await import("./loader-extensions.js")
-                : // @ts-expect-error
-                    await import("./loader.js");
+            MainExport = await import("./loader.js");
         }
         Mod.export = await MainExport.default();
     },
@@ -8534,6 +8526,7 @@ const KEYBOARD_MAP = {
     ".": ImGuiKey._Period,
     "/": ImGuiKey._Slash,
     ";": ImGuiKey._Semicolon,
+    ":": ImGuiKey._Semicolon,
     "=": ImGuiKey._Equal,
     "[": ImGuiKey._LeftBracket,
     "\\": ImGuiKey._Backslash,
